src/components/sidebar/right-sidebar/Filters.tsx

```diff
@@ -39,15 +40,23 @@ export default function Filters({ page }: FilterProps) {
         });
         break;
       default:
-        options.forEach((o: string) => {
-          const translatedString = Translate(jsonObjectString, [toPascalCase(o)]);
-          const alternativeString = Translate(jsonObjectString, [o.replace(/ /g, '')]);
-          let text = !alternativeString && !translatedString ? o + "*" : (translatedString ? translatedString : alternativeString);
-          formatted_options.push({
-            key: o,
-            text: text,
-            value: o
-          })
+        options.forEach((o: string | string[]) => {
+          if (filter_type === "population_group") {
+            formatted_options.push({
+              key: state.language === LanguageType.english ? o[0] : o[1],
+              text: state.language === LanguageType.english ? o[0] : o[1],
+              value: state.language === LanguageType.english ? o[0] : o[1]
+            })
+          } else {

```

Comment:
I think eventually we should be deprecating this code block as the whole filter options translation mechanism should be standardized.

If you agree, can you make a comment that this will be deprecated soon! And make a ticket to move all other translations for options to the backend!